### ISSUE TITLE
Invalid/missing `aria-disabled` property when all ribbon buttons in a group are disabled

### ISSUE THREAD

**Issue by tobz1000** [OfficeDev/office-js#5385]
When a single ribbon button for an addin is disabled, the `aria-disabled="true"` property is added to the button element.

When all buttons in a group within a ribbon toolbar are disabled, the `aria-disabled` property is moved to an ancestor element:
![Image](https://github.com/user-attachments/assets/5b3558ff-4948-4880-960a-6257bff1ba25)

However, this element is a plain div with no role, and so is strictly invalid for the `aria-disabled` property. From [the spec](https://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled):

> This state is being deprecated as a global state in ARIA 1.2. In future versions it will only be allowed on roles where it is specifically supported.

It would be better to move the `aria-disabled` property to the `role="group"` element in this scenario.

This issue impacts my ability to run UI tests using Playwright, which currently ignores the property on invalid elements, so I cannot assert that the buttons are disabled.

**Comment by alison-mk** [comment]
Hi @tobz1000, thank you for submitting this issue. @EsterBergen could you take a look at this ribbon button issue? 

Cheers,
Alison
